cast-receiver/scss/_typography.scss

```diff
@@ -0,0 +1,73 @@
+@import "colors";
+
+body {
+  color: $foreground-secondary;
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 1rem;
+
+  .title {
+    color: $foreground-primary;
+    font-size: 3rem;
+
+    span.subtitle {
+      margin-left: 0.5rem;
+      font-size: 2rem;
+      font-weight: 400;
+    }
+  }
+
+  .caption {
+    font-size: 1.5rem;
+  }
+}
+
+/* latin-ext */
+@font-face {
+  font-family: "Source Sans Pro";
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: local("Source Sans Pro Regular"), local("SourceSansPro-Regular"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=5670181)